src/index.ts

```diff
@@ -1,8 +1,17 @@
 import type { Channel } from "@storybook/channels";
+import { readConfig, writeConfig } from "@storybook/csf-tools";
+import { exec } from "child_process";
 // eslint-disable-next-line import/no-unresolved
 import { run } from "chromatic/node";
+import { promisify } from "util";
 
-import { BUILD_STARTED, START_BUILD } from "./constants";
+import { BUILD_STARTED, START_BUILD, UPDATE_PROJECT, UpdateProjectPayload } from "./constants";
+import { findConfig } from "./utils/storybook.config.utils";
+
+const {
+  CHROMATIC_BASE_URL = "https://www.chromatic.com",
+  CHROMATIC_ADDON_NAME = "@chromaui/addon-visual-tests",
+} = process.env;
```

Comment:
I recommend using globals over `process.env`.

But if you're going to use `process.env`, please use the full path, and don't destructure:
```
const CHROMATIC_BASE_URL = process.env.CHROMATIC_BASE_URL || 'https://chromatic.com';
```

This way you won't get the entire `process.env` stringified into your code (bloat).

Reply:
This is `index.js` (a preset) it isn't replaced.

Reply:
> I recommend using globals over process.env.

Do we have a way to do this in a builder agnostic way?

Reply:
Ow, whoops, I didn't catch this will be run in node when I was reviewing it.. I'll discard this recommendation!